pkg/controller/controller_serviceclass.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Should we make the copywrite on these 2018? It's technically a new file.

Reply:
+1, updating!